scripts/sct_testing.py

```diff
@@ -232,7 +232,7 @@ def main(args=None):
     param.path_data = os.path.abspath(param.path_data)
 
     # check existence of testing data folder
-    if not os.path.isdir(param.path_data) or param.download:
```

Comment:
the reason we forced download if folder exists is because sometimes the data are altered (new version, processed inputs), which could affect the integrity of testing. I remember having spent 1 hour on a bug, after realizing that the bug was caused by corrupted testing data (which were not re-downloaded). So if user asks for download, i would just download, regardless if the folder already exists

Reply:
Ah okay so there is a good reason. Thanks. I'll close this.

Reply:
maybe no need to close this? the changes in CLI flags are relevant, no?

Reply:
I split it up, that's over in #/2672.